logic/common/tiled_utils.py

```diff
@@ -164,6 +164,22 @@ def RotateTileId(tile_id):
 #--------------------------------------------------#
 '''Object Properties'''
 
+def GetProperty( curr_obj, property_name ):
```

Comment:
I actually think it's best NOT to put it inside `level_playdo.py`. If we are to place it next to `_AddPropertiesToObject`, i.e. outside the class definition, it would not be callable without importing the playdo file, which we normally don't outside of CLI.

But if we are to place the function inside class definition, it would then become inaccessible in one of the functions in `conflict.py`, where the playdo is not passed down. I'm sure there might be more instances of that in the future too, and having to pass down playdo each time just feels like an inelegant solution?

Overall, I think keeping this function in one of the "utility files" is the simplest solution, since housing publicly accessible utility functions is exactly what they're for. In contrast, `playdo.py` is a "class file", which I believe should contain the class as the only publicly accessible thing.